tests/e2e/manifest/values-istio-ingress.yaml

```
@@ -0,0 +1,26 @@
+      targetPort: 8080
+      name: http2
+    - port: 443
+      targetPort: 8443
+      name: https
+    - port: 31400
+      targetPort: 31400
+      name: tcp
+      # This is the port where sni routing happens
+    - port: 15443
+      targetPort: 15443
+      name: tls
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)